test/unit/format_routing_constraint_test.rb

```diff
@@ -21,7 +21,7 @@ def request
   end
 
   def artefact(format)
-    stub(format: format)
+    stub(:[] => format)
```

Comment:
The `GdsApiAdapters::Response` object that is being doubled in this test has changed. It no longer implements a `method_missing` approach to interrogating the underlying JSON data, but instead gives us the [square brackets method as a way of querying an internal Hash of parsed data.](https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/response.rb#L24)